music_album.rb

```diff
@@ -0,0 +1,13 @@
+class MusicAlbum
```

Comment:
[REQUIRED] Kindly ensure that `MusicAlbum` inherits from the `item` class. This is because there should be a parent-child relationship between them.